packages/routing-utils/src/types.ts

```diff
@@ -123,5 +123,5 @@ export interface AppendRoutesToPhaseProps {
   /**
    * The phase to append the routes such as `filesystem`.
```

Comment:
Might be worth expanding this comment with an explanation of the behaviour of passing `null`